### PR TITLE
CO-2283_Lightbox_View_missing_from_OIS_Name_entry_in_canvas

### PR DESCRIPTION
**Pull request by ioigoume**
Use lightbox to render readonly Name entry linked to OIS in canvas

### CODE REVIEW

File: app/View/Names/fields.inc [hunk 1]
```diff
@@ -171,7 +171,7 @@
             print $this->Form->error('Name.type');
           }
         } else {
-          print $vv_available_types[ $names[0]['Name']['type'] ];
+          print _txt('en.name.type', null, $names[0]['Name']['type']);
```
Comment (boshrin) [inline app/View/Names/fields.inc:174]: What's the reason for this change? `$vv_available_types` is the "newer" way to pass localization strings.

Comment (ioigoume) [reply app/View/Names/fields.inc]: Reverted the change

**Comment by boshrin** [comment]
I'm going to merge this change as is. Let's take the `$vv_available_types` discussion to a separate issue.
